sormas-api/src/main/java/de/symeda/sormas/api/event/EventActionCriteria.java

```diff
@@ -0,0 +1,63 @@
+/*******************************************************************************
+ * SORMAS® - Surveillance Outbreak Response Management & Analysis System
+ * Copyright © 2016-2018 Helmholtz-Zentrum für Infektionsforschung GmbH (HZI)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <https://www.gnu.org/licenses/>.
+ *******************************************************************************/
+package de.symeda.sormas.api.event;
+
+import java.io.Serializable;
+
+import de.symeda.sormas.api.BaseCriteria;
+import de.symeda.sormas.api.action.ActionCriteria;
+import de.symeda.sormas.api.action.ActionStatus;
+import de.symeda.sormas.api.utils.IgnoreForUrl;
+
+/**
+ * Criteria filter for actions related to en event.
+ */
+public class EventActionCriteria extends BaseCriteria implements Serializable {
```

Comment:
Any reason to specifically call this EventActionCriteria and not only ActionCriteria?

Reply:
The initial request planned to integrate this management of actions on other contexts (case, contact,…), so I tried to separate what is specific to actions on events.
I didn't want to have `@IgnoreUrl` directly in `ActionCriteria` as it's specific to event pages.
Thinking back on it now, I'm not convinced of the usefulness of `EventActionCriteria`.